package.json

```
@@ -14,7 +14,7 @@
     "clean": "watchman watch-del-all && rm -rf /tmp/metro-bundler-cache-* && rm -rf /tmp/haste-map-react-native-packager-* && rm -rf node_modules/ && yarn cache clean --force",
     "commit": "commit-wizard",
     "ios": "yarn run build-rust-ios && react-native run-ios",
-    "lint": "eslint",
+    "lint": "./node_modules/.bin/eslint . --fix",
```

what about `npx eslint`. Looks cleaner to me.

```suggestion
    "lint": "npx eslint . --fix",
```

---

Unrelated to that, I wouldn't expect that calling `lint` would trigger a `fix` by default, that should rather be another script, like `(npm run|yarn) lint:fix`.

---

^ +1, seems the "fix" is getting tripped up with the slashes in `suri.spec.js` and making the build fail.

---

I think `npx` wouldn't work in CI, they do not have global context. I will use `lint:fix` for that

---

So true, updated!